wled00/const.h

```diff
@@ -232,6 +232,7 @@
 #define TYPE_WS2811_400KHZ       24            //half-speed WS2812 protocol, used by very old WS2811 units
 #define TYPE_TM1829              25
 #define TYPE_UCS8903             26
+#define TYPE_FW1906              28            //RGB + CW + WW + unused channel (6 channels per IC)
```

Comment:
I think the ordering here is an incorrect assumption that impacts the flexibility of this PR>

FW1906 is just a dual RGB (or, more accurately, BGR) IC — there aren't specific CW/WW pins.

It's up to the manufacture how to connect their LEDs, and unfortunately, they take liberties here.

The FW1906 strip I have, for instance, orders its channels as WW, null, G R B, CW. This ordering, while strange and annoying, actually does make sense — it lines up nicely with the physical ordering of the LEDs on the strip and optimized for the traces vs how humans want to think about it.

I'd think before this PR is super viable to merge, we need the ability to fully customize the channel orders like we can in RGB.

Reply:
Hey there shnhrrsn. I ordered 200 FW1906 ICs and have designed a driver board utilizing this IC. My only problem is getting the firmware working on my ESP32. I have done the whole npm run but i always get an error. What steps did you do to get this working or do you have a firmware build you could share so i can test my boards. Cheers mate

Reply:
@oakey81 This thread is about line 235 in const.h, and color ordering of FW1906 chips.
please don't drift off-topic.

Reply:
Yeah mate i get that but I was just asking how he actually got the firmware compiled properly and working. As he stated he got it working. Anyway this whole thread jumps form one issue to the next if you scroll back a fair way.

Reply:
we call that "code review". In contrast to "support request" ;-)